tronweb/coffee/models.coffee

```diff
@@ -54,6 +54,20 @@ class window.RefreshModel extends Backbone.Model
             console.log("scheduled with " + @interval)
             @timeout = setTimeout(@doRefresh, @interval)
 
+class window.StatusModel extends Backbone.Model
+
+    initialize: (options) =>
+        super options
+        options = options || {}
+
+    urlRoot: ->
+        "/status/"
+
+    parse: (resp, options) =>
+        booted = moment.unix(resp['boot_time']).format()
+        uptime = moment.duration(moment() - booted).minutes()
+        $('#version').html('Tron: <b>v' + resp['version'] + '</b>  Uptime: <b>' + uptime + 'm</b> Boot: <b>' + booted + '</b>')
```

Comment:
I think it would be easier to read if you bold the keys (Tron, Boot) instead of the values.
Also, I wrote uptime in the ticket, but I think boot time is easier to use, and we probably don't need both?

Reply:
Sure, I removed uptime and tried simplifying the format of the boot time.